packages/quantic/force-app/main/default/lwc/quanticFacet/e2e/fixture.ts

```diff
@@ -0,0 +1,71 @@
+import {FacetObject} from './pageObject';
+import {quanticBase} from '../../../../../../playwright/fixtures/baseFixture';
+import {useCaseEnum} from '../../../../../../playwright/utils/useCase';
+import {InsightSetupObject} from '../../../../../../playwright/page-object/insightSetupObject';
+import {
+  searchRequestRegex,
+  insightSearchRequestRegex,
+} from '../../../../../../playwright/utils/requests';
+import {SearchObjectWithFacet} from '../../../../../../playwright/page-object/searchObjectWithFacets';
+import facetData from './data';
+
+const pageUrl = 's/quantic-facet';
+
+interface FacetOptions {
+  value?: string;
+  caption?: string;
+}
+
+type QuanticFacetE2EFixtures = {
```

Comment:
Im curious here, did you investigate the possibility to have a generic "baseFacetFixture" or something like that to avoid duplicating setup code when we tackle the tickets for the other types of facets? Any consideration we should account for when working on these tickets?

Reply:
Was going to add this refactoring in the next facet PR I will make to avoid overloading this PR even more but here you go!

Basically added a new BaseFacet Fixture and a BaseFacet Object.
The BaseFacet Object will contain all the logic related to check the analytics and the search requests made from the different facet components.
This new BaseFacet Fixture and a BaseFacet Object will be used in all the other facet types.